### PR TITLE
[VEN-2902] Add EIGEN Market

### PR DESCRIPTION
**Pull request by web3rover**
## Description

Resolves VEN-2902

### CODE REVIEW

File: vips/vip-389/bsctestnet.ts [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+      {
+        target: WETH_PRIME_CONVERTER,
+        signature: "setConversionConfigs(address,address[],(uint256,uint8)[])",
+        params: [BaseAssets[3], [EIGEN], [[0, 1]]],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: XVS_VAULT_CONVERTER,
+        signature: "setConversionConfigs(address,address[],(uint256,uint8)[])",
+        params: [BaseAssets[4], [EIGEN], [[0, 1]]],
```
Comment (chechu) [inline vips/vip-389/bsctestnet.ts:128]: The conversion incentive should be 1e14 for every asset from now on. Not critical for sepolia, but relevant for mainnet

File: simulations/vip-389/sepolia.ts
Comment (chechu) [inline simulations/vip-389/sepolia.ts:1]: Would it make sense to also add a simulation for the VIP on BNB Chain, checking the bridged commands, for example?

**Review by chechu — COMMENTED** [review]
Sepolia VIP lgtm

**Comment by chechu** [comment]
Sepolia: https://testnet.bscscan.com/tx/0xde0347163b1259ea955c3e83ffd9de7d856e9adbacc4f1b84cd24a06c05a0aaa

**Review by chechu — COMMENTED** [review]
Pending points:
* Bootstrap liquidity in the Treasury (and update the simulation block)
* Refund the bootstrap liquidity provider with USDC on BNB Chain

File: vips/vip-393/bscmainnet.ts [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+          ],
+        ],
+        dstChainId: LzChainId.ethereum,
+      },
+
+      // Add Market
+      {
+        target: ethereum.VTREASURY,
+        signature: "withdrawTreasuryToken(address,uint256,address)",
+        params: [EIGEN, INITIAL_SUPPLY, ethereum.GUARDIAN],
```
Comment (chechu) [inline vips/vip-393/bscmainnet.ts:124]:
```suggestion
        params: [EIGEN, INITIAL_SUPPLY, ethereum.NORMAL_TIMELOCK],
```

File: simulations/vip-393/ethereum.ts [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+    it("check protocol share reserve", async () => {
+      expect(await vEIGENContract.protocolShareReserve()).equals(PROTOCOL_SHARE_RESERVE);
+    });
+
+    it("check reserve factor", async () => {
+      expect(await vEIGENContract.reserveFactorMantissa()).equals(parseUnits("0.25", 18));
+    });
+
+    it("check protocol seize share", async () => {
+      expect(await vEIGENContract.protocolSeizeShareMantissa()).equals(parseUnits("0.05", 18));
```
Comment (GitGuru7) [inline simulations/vip-393/ethereum.ts:110]: Shouldn't protocol seize share be 1e16?

File: vips/vip-393/bscmainnet.ts (hunk 2 above)
Comment (chechu) [reply vips/vip-393/bscmainnet.ts]: Not needed anymore

File: simulations/vip-393/ethereum.ts (hunk 3 above)
Comment (chechu) [reply simulations/vip-393/ethereum.ts:113]: The "protocol seize share" must be 5e16, as it is in the rest of the markets in the Core pool. The spect was wrong

**Review by chechu — COMMENTED** [review]
Mainnet commands lgtm

**Review by kkirka — APPROVED** [review]
Mainnet commands and risk parameters LGTM

**Review by chechu — APPROVED** [review]

**Pull request merged**